nodeup/pkg/model/kube_apiserver.go

```diff
@@ -318,6 +352,42 @@ func (b *KubeAPIServerBuilder) buildPod() (*v1.Pod, error) {
 
 	pod.Spec.Containers = append(pod.Spec.Containers, *container)
 
+	if b.Cluster.Spec.EncryptionConfig != nil && *b.Cluster.Spec.EncryptionConfig {
+		sideCar := &v1.Container{
+			Name:  "kms-plugin",
+			Image: "sethpollack/kmsplugin:latest",
+			Args: []string{
+				"-listen",
+				"/var/run/kmsplugin/socket.sock",
+				"-key",
+				keyName,
+			},
+			LivenessProbe: &v1.Probe{
+				Handler: v1.Handler{
+					HTTPGet: probeAction,
```

Comment:
I think we want to use our own probe, instead of reusing the apiserver probe, because the apiserver probe could vary if we have https mode turned on